pkg/ebpf/tracee.go

```diff
@@ -1736,3 +1745,34 @@ func (t *Tracee) ready(ctx gocontext.Context) {
 func (t *Tracee) triggerMemDumpCall(address uint64, length uint64, eventHandle uint64) error {
 	return nil
 }
+
+// SubscribeAll returns a stream subscribed to all policies
+func (t *Tracee) SubscribeAll() *streams.Stream {
+	return t.subscribe(policy.AllPoliciesOn)
+}
+
+// Subscribe returns a stream subscribed to selected policies
+func (t *Tracee) Subscribe(policyNames []string) (*streams.Stream, error) {
+	var policyMask uint64
```

Comment:
Not directly related to this PR, but it might be nice if we will define a new type for the policy mask instead of using uint64. This way, it will be easier to modify the number of policies we have, if and when we will decide to do it.